Documentation/CRDs/Cluster/ceph-cluster-crd.md

```diff
@@ -542,6 +543,7 @@ If a user configures a limit or request value that is too low, Rook will still r
 * `crashcollector`: 60MB
 * `mgr-sidecar`: 100MB limit, 40MB requests
 * `prepareosd`: no limits (see the note)
+* `exporter`: 512MB
```

Comment:
This list is more of a recommended limit, rather than an upper limit. So based on your testing, hopefully we can lower this recommended limit maybe to 256 or even 128Mi